lib/endpoints/class-wp-rest-comments-controller.php

```diff
@@ -282,8 +282,18 @@ public function delete_item( $request ) {
 			}
 
 			$result = wp_trash_comment( $comment->comment_ID );
+			$status = 'trashed';
 		}
 
+		$response = $this->prepare_item_for_response( $comment, $get_request );
```

Comment:
If we do this _after_ the comment has been deleted, I think it's possible some preparation might fail here?